components/automate-ui/src/app/pages/+compliance/+reporting/reporting.component.ts

```diff
@@ -339,6 +347,12 @@ export class ReportingComponent implements OnInit, OnDestroy {
       } else {
         typeName = 'control_name';
       }
+    } else if (this.isTypeControlTag(type.name)) {
+      if (value.title === 'no value') {
+        typeName = type.name;
+        filterValue = value.id;
+        this.reportQuery.setFilterTitle(typeName, value.id, value.title);
+      }
```

Comment:
This lives inside the function where we build our URL Parameters when adding a new filter.  This is where we've decided custom alterations will live.

this function checks if the current filter is 3rd level, i.e. `control_type:something`, and if it is, it sets the filter value to an empty string, which is what the API endpoint needs to understand "no values"